python/tirex_tracker/__init__.py

```diff
@@ -966,15 +1039,15 @@ def tracking(
     poll_intervall_ms: int = -1,
     system_name: Optional[str] = None,
     system_description: Optional[str] = None,
-    export_file_path: Optional[PathLike] = None,
+    export_file_path: Optional[PathLike] | str = None,
     export_format: Optional[ExportFormat] = None,
 ) -> TrackingHandle:
     return TrackingHandle.start(
         measures=measures,
         poll_intervall_ms=poll_intervall_ms,
         system_name=system_name,
         system_description=system_description,
-        export_file_path=export_file_path,
+        export_file_path=None if export_file_path is None else Path(export_file_path),
```

Comment:
I would not convert the `PathLike` to a `Path` here just yet. It should be `PathLike` in all user-facing functions and can then be converted to a `Path` only where needed, e.g., in https://github.com/tira-io/tirex-tracker/pull/46/files#diff-f5afdbe9855237f08a8d1cf80146de6117c3dcfd29d76f32346a48d3b48de1a3R489-R494